tests/lib/c_lib/common/testcase.yaml

```diff
@@ -4,6 +4,11 @@ common:
   ignore_faults: true
   integration_platforms:
     - mps2/an385
+  platform_exclude:
+    # CONFIG_MMU=y but no arch_mem_map() or arch_mem_unmap()
```

Comment:
hmm, if this is the only way possible, sure, but I hope we can find a solution without explicit platform excludes...

Reply:
Same - there are a couple of places where we have these excludes, and I would prefer to exclude the excludes.

Reply:
If this is the same build failure, then I think the simplest workaround would be to implement an `arch_mem_map()` and `arch_mem_unmap()` for these platforms that simply fails.